cmd/influxd/run/config.go

```diff
@@ -120,12 +121,20 @@ func NewDemoConfig() (*Config, error) {
 	return c, nil
 }
 
+// trimBOM trims the Byte-Order-Marks from the beginning of the file.
+// this is for Windows compatability only.
+// see https://github.com/influxdata/telegraf/issues/1378
+func trimBOM(f []byte) []byte {
+	return bytes.Trim(f, "\xef\xbb\xbf")
```

Comment:
This should definitely be `bytes.TrimPrefix`... `bytes.Trim` will remove all of those bytes from the file